jargit.py

```diff
@@ -243,7 +245,7 @@ def checkout(parsed_args) -> None:
 
 
 _SCRIPT_NAME: Optional[str] = None
-_SCRIPT_USAGE: Optional[str] = '\n'.join(('jargit.py command [command_args ...] [-h] [-v | -q] [--interactive]',
+_SCRIPT_USAGE: Optional[str] = '\n'.join(('jargit command [command_args ...] [-h] [-v | -q] [--interactive]',
```

Comment:
Because I added a `jargit` command on the server I removed the extension from the help information.